src/beautifiers/formatR/index.coffee

```diff
@@ -0,0 +1,25 @@
+###
+Requires [formatR](https://github.com/yihui/formatR)
+###
+path = require("path")
+
+"use strict"
+Beautifier = require('../beautifier')
+
+module.exports = class R extends Beautifier
+  name: "R beautifier"
```

Comment:
`package.json` and `README.md` are automatically generated. `README.md` from [`README-template.md`](https://github.com/Glavin001/atom-beautify/blob/master/README-template.md). You do not have to manually change those as long as the Languages and Beautifiers are setup properly. Although it will not remove `keywords` so it was good of you to remove `r beautifier`: https://github.com/Glavin001/atom-beautify/blob/master/docs/index.coffee#L194